client/me/account/main.jsx

```diff
@@ -492,7 +502,8 @@ const Account = React.createClass( {
 						id="url"
 						name="url"
 						onFocus={ this.recordFocusEvent( 'Web Address Field' ) }
-						valueLink={ this.valueLink( 'user_URL' ) }
+						value={ this.getUserSetting( 'user_URL' ) || '' }
+						onChange={ this.updateUserSettingInput( 'user_URL' ) }
```

Comment:
> I'm not sure the commit with these changes came through…

I squashed these changes into the "Removed usage of valueLink from client/me/account" and did `push -f`. The Github UI is showing me the right version now.